files/usr/etc/profile.d/ls.tcsh

```diff
@@ -6,6 +6,8 @@ if ( -x /usr/bin/dircolors ) then
 	eval `/usr/bin/dircolors -c $HOME/.dir_colors`
     else if ( -r /etc/DIR_COLORS ) then
 	eval `/usr/bin/dircolors -c /etc/DIR_COLORS`
+    else if ( -r /usr/etc/DIR_COLORS ) then
+	eval `/usr/bin/dircolors -c /usr/etc/DIR_COLORS`
```

Comment:
What's the advantage? There's only one LS_COLORS so one call would just override the next. Anyway I'm just following the existing logic here. It doesn't read `/etc/DIR_COLORS` and then `$HOME/.dir_colors` to override it either.

Reply:
Hmmm ... the idea behind would be that if the admin changes one color for an entry or add an new one only this could be part of `/etc/DIR_COLORS`.  Currently the onyl way would be copy `/usr/etc/DIR_COLORS` to `/etc/DIR_COLORS` andthen do the admins changes

Reply:
yes. fixing that would be nice but kind of out of scope here IMO.  After all the same inconvenience exists with `~/.dir_colors` today. Also when thinking about fixing that one could just as well make `ls` read those files directly and avoid the use of the external `dircolors` command.